web/src/settings_components.ts

```diff
@@ -1044,7 +1044,7 @@ export function populate_data_for_realm_settings_request(
                     continue;
                 }
 
-                const realm_group_settings_using_new_api_format = new Set([
+                const realm_group_settings = new Set([
```

Comment:
Can we compute this list from `realm.server_supported_permission_settings`? It would be reasonable to have a function in `group_permission_settings.ts` that exposes this list.

(Would save a bit of manual work every time we add a new setting).